app/controllers/auth_key_pair_cloud_controller.rb

```diff
@@ -56,11 +56,10 @@ def show
       @showtype = "main"
       set_summary_pdf_data if %w(download_pdf summary_only).include?(@display)
     when "instances"
-      table = @display == "vm_cloud"
```

Comment:
I am not sure about the `@display` variable here. I am afraid that it's one of those variable that can be used basically anywhere and it might not be safe to just not set it.